compose/ui/ui-text/src/skikoMain/kotlin/androidx/compose/ui/text/platform/SkiaParagraph.skiko.kt

```diff
@@ -493,11 +494,34 @@ internal class ParagraphBuilder(
         return null
     }
 
+    private fun makeSkFontRasterizationSettings(style: TextStyle): SkFontRastrSettings {
+        val rasterizationSettings = style.paragraphStyle.platformStyle?.fontRasterizationSettings
+            ?: FontRasterizationSettings.defaultForCurrentPlatform
+        val edging = when (rasterizationSettings.smoothing) {
+            FontSmoothing.None-> org.jetbrains.skia.FontEdging.ALIAS
+            FontSmoothing.AntiAlias -> org.jetbrains.skia.FontEdging.ANTI_ALIAS
+            FontSmoothing.SubpixelAntiAlias -> org.jetbrains.skia.FontEdging.SUBPIXEL_ANTI_ALIAS
+        }
```

Comment:
1. Apply auto formatting
2. it's a good candidate for extraction to internal extension function for this enum (I'd put it in `FontRasterizationSettings.skiko.kt`) file